### PR TITLE
[WIP] port to PyQt5

### LINKED ISSUE #662: Port the client to PyQt5
Arch Linux dropped QtWebkit from the `python2-pyqt4` package. Probably because it's an ancient security nightmare. This forces me to hold back package upgrades. `python2-pyqt5`contains QtWebkit though. Porting should be simple, but touches a lot of code.

After ICE™️

### PR DESCRIPTION
**Pull request by muellni**
Fix #662 

To do:
- [ ] Figure out how to remove the empty progress dialog on startup
- [ ] Fix double window title bar (not on Windows)
- [x] Build and test windows installer
- [ ] Test every corner of the client :)
- [ ] Figure out how to get PyQt5 for Python 2 installed on travis (maybe use conda: https://conda.io/docs/travis.html )
- [ ] Fix hard crash on Windows after some seconds

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This is the source code for the FA Forever Lobby.
 Pre-requisites are:
 
 - Python 2.7.x
-- PyQt4 4.7+
+- PyQt5 4.7+
```
Comment (Wesmania) [inline readme.md]: 4.7 -> ??

File: appveyor.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ install:
   - ps: "$env:faf_version = \"$(&C:\\Miniconda\\python .\\src\\config\\version.py)\".Replace(' ','')"
   - ps: "$env:APPVEYOR_BUILD_VERSION = \"$($env:faf_version)+$($env:APPVEYOR_BUILD_NUMBER)\""
   - ps: "Write-Host \"$($env:faf_version) + $($env:APPVEYOR_BUILD_NUMBER) = $($env:APPVEYOR_BUILD_VERSION)\""
-  - ps: "$env:PYTEST_QT_API=\"pyqt4v2\""
+  - ps: "$env:PYTEST_QT_API=\"PyQt5v2\""
```
Comment (Wesmania) [inline appveyor.yml]: This and other places will need an s@PyQt5v2@PtQt5@

Comment (muellni) [reply appveyor.yml]: Yes, thanks for looking at this PR. It certainly is still a lot of work. I started this to see if it can be done.

**Comment by muellni** [comment]
#509

**Pull request closed without merging**